docs/components/docs/Navigation.tsx

```diff
@@ -144,6 +144,12 @@ export function DocsNavigation() {
         <NavItem href="/docs/guides/custom-fields">
           Custom Fields <Badge look="success">New</Badge>
         </NavItem>
+        <NavItem href="/docs/guides/custom-admin-ui-logo">
+          Custom Admin UI Logo <Badge look="success">New</Badge>
+        </NavItem>
+        <NavItem href="/docs/guides/custom-admin-ui-pages">
+          Custom Admin UI Pages <Badge look="success">New</Badge>
+        </NavItem>
```

Comment:
```suggestion
```

Also let's leave the greyed out link to `Custom Admin UI Pages` below until we've finished that

Reply:
ARGH i thought i removed this, ty for the catch